tools/quota-monitoring-alerting/java/README.md

```diff
@@ -452,7 +452,44 @@ Quota monitoring reports can be scheduled from the Data Studio dashboard using 
 
 <img src="img/datastudio_schedule_email.png" align="center" />
 
-## 4. What is Next? 
+
+### 4. Alerting
+The alerts about services nearing their quota limits can be configured to be sent via email as well as following external services:
+- Slack
+- PagerDuty
+- SMS
+- Custom Webhooks
+
+##### Slack Configuration
+To configure notifications to be sent to a Slack channel, you must have the Monitoring NotificationChannel Editor role on the host project.
+###### Create Notification Channel
+1. In the Cloud Console, use the project picker to select your Google Cloud project, and then select Monitoring, or click the link here: Go to Monitoring
+2. In the Monitoring navigation pane, click  Alerting.
+3. Click Edit notification channels.
+4. In the Slack section, click Add new. This brings you to the Slack sign-in page:
+&nbsp;&nbsp; a.  Select your Slack workspace.
```

Comment:
These lines will not render properly due to the non-breaking space.

Please have a look at github doc for how to indent sub-lists

https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists